src/main/java/sqlline/Commands.java

```diff
@@ -947,7 +950,17 @@ private void execute(String line, boolean call, DispatchCallback callback) {
 
       try {
         long start = System.currentTimeMillis();
-
+        String pattern = sqlLine.getOpts().getConfirmPattern();
+        if (Pattern.compile(pattern).matcher(sql).find()
+              && sqlLine.getOpts().getConfirm()) {
+          String question = sqlLine.loc("really-perform-action");
+          final int userResponse = getUserAnswer(question, 'y', 'n');
```

Comment:
Potential answers can be 'Y' and 'N' (uppercase only)

Reply:
Nothing specific, its just that it introduces a small delay since we need to put a thought for upper case letters. Its a good to have thing, I am fine with either.